config/webpack.config.js

```diff
@@ -28,21 +28,18 @@ var config = {
   },
 
   resolve: {
-    extensions: ['', '.js'],
+    extensions: ['', '.js', '.jsx'],
     root: path.join(__dirname, '..', 'webpack')
   },
 
   module: {
     loaders: [
       {
-        test: /\.js$/,
+        test: /\.jsx*$/,
```

Comment:
we've stopped using the .jsx extension. the test has been changes to reflect this.